pipelines/cohort_screening/src/requirements.txt

```diff
@@ -0,0 +1,5 @@
+gtfparse
+jinja2
+pandas
```

Comment:
Do we want to version these requirements?

Reply:
done